spec/rubocop/comment_config_spec.rb

```diff
@@ -61,7 +61,7 @@
     end
 
     def disabled_lines_of_cop(cop)
-      (1..source.size).each_with_object([]) do |line_number, disabled_lines|
+      (1..source.lines.count).each_with_object([]) do |line_number, disabled_lines|
```

Comment:
Was bug in the test. `source.size` is character count of source, not line count.

Reply:
👍